gigadb/app/tools/wasabi-migration/README.md

```diff
@@ -295,4 +328,20 @@ This can be overridden using the `--max-batch-size`. For example, to increase
 the batch size to 200:
 ```
 $ docker-compose run --rm -e HOST_HOSTNAME=`hostname` rclone_cngb /app/rclone_copy.sh --starting-doi 100000 --ending-doi 100300 --max-batch-size 300
-```
\ No newline at end of file
+```
+
+#### Testing the notification feature if error occurs during the backup process
+```
+# To generate log file containing ERROR
+[gigadb@cngb-gigadb-bak wasabi-migration]$ docker-compose run --rm rclone_cngb /app/rclone_copy.sh --starting-doi 100002 --ending-doi 100320 
+# Check the log file can be found in the logs/ dir
+# Spin up the swatchdog 
+[gigadb@cngb-gigadb-bak wasabi-migration]$ docker-compose up -d rclone_cngb 
+# Stop and remove rclone container 
+[gigadb@cngb-gigadb-bak wasabi-migration]$ docker stop $(docker ps -aq) && docker rm $(docker ps -aq)
+# Or chain the cmd together, so the swatchdog could recognize the latest log and capture it, and then send it gitter room
+[gigadb@cngb-gigadb-bak wasabi-migration]$ docker-compose run --rm rclone_cngb /app/rclone_copy.sh --starting-doi 100002 --ending-doi 100320 && docker-compose up -d rclone_cngb && docker stop $(docker ps -aq) && docker rm $(docker ps -aq)
+Creating wasabi-migration_rclone_cngb_1 ... done
```

Comment:
I am able to manually execute these commands to see the functionality working. I wonder if these commands would be better placed in a bash script for execution? The bash script would contain logic to check if a Swatchdog container is already running and, if so, then could use `docker restart` to get it to monitor the latest rclone_copy log file. If not, then the script would spin up a new Swatchdog container. I also wonder if it is a good idea to use `&&` to chain the commands together. If `rclone_copy` is long running then Swatchdog would not run until `rclone_copy` process has finished which could take days? Probably best that the Swatchdog container starts monitoring as soon as the new log file is present.